shopfloor/actions/data.py

```diff
@@ -214,3 +214,54 @@ def picking_batches(self, record, with_pickings=False, **kw):
     @property
     def _picking_batch_parser(self):
         return ["id", "name", "picking_count", "move_line_count", "total_weight:weight"]
+
+    def picking_type(self, record, **kw):
+        parser = self._picking_type_parser
+        return self._jsonify(record, parser, **kw)
+
+    def picking_types(self, record, **kw):
+        return self.picking_type(record, multi=True)
+
+    @property
+    def _picking_type_parser(self):
```

Comment:
@sebalix it's probably better to add a computed fields for this or a single Serialized computed fields because you'll save some queries (you get 4 now).